yaml/shared/src/main/scala/org/virtuslab/yaml/YamlError.scala

```diff
@@ -17,6 +17,8 @@ object ParseError:
         |${got.pos.errorMsg}""".stripMargin
   )
   def from(expected: TokenKind, got: Token): ParseError = ParseError.from(expected.toString, got)
+  def from(expected: List[TokenKind], got: Token): ParseError =
+    ParseError.from(expected.toString, got)
```

Comment:
It could be confusing if we return List[TokenKind].toString, it should be more precise.